src/js/facility-locator/components/FacilityDetailsInfoPhone.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { Component, PropTypes } from 'react';
+
+class FacilityDetailsInfoPhone extends Component {
+  appendPhone(numbers) {
+    const str = numbers.map((number) => { return `${number}<br />`; }).join('');
+
+    return { __html: str };
+  }
+
+  render() {
+    return (
+      <div>
+        <h5 style={{ marginTop: '15px' }}>Phone:</h5>
+        <table style={{ marginTop: '-10px', border: 'none' }} cellPadding="0" cellSpacing="0">
+          <tr>
+            <td>Main Number:</td>
+            <td dangerouslySetInnerHTML={this.appendPhone(this.props.main)}/>
```

Comment:
Rather than setting innerHTML, I would prefer to see something like this where you split them as block level elements:

```
const mainNumbers = this.props.main.map(number => <p>{number}</p>)
...
<td>{mainNumbers}</td>
```